Edit Pathways

amontes261 opened the pull request:
Resolves #2. Branch can be deleted after merge.

amontes261 committed:
bruh

y my code no work

nishi7409 committed:
Merge branch 'master' into 2/Alex

amontes261 committed:
Added Pathway Editing Feature

Added:
	- iCloud Drive Synchronization file to .gitignore
	- Header to let user know when they’re editing a pathway
	- Actions for the save button that restore from the editing-state

Changes:
	- Made font bigger on My Pathways page for indicator that appears when no Pathways have been added

Fixed:
	- Edit Pathway button: works properly now!

Misc:
	- Commented-out code that’s meant to refresh page on home-logo press –– will be implemented in future [ location.reload( ) ]

	==  RESOLVES ISSUE #2 ==

nishi7409 commented:
It's not intuitive as to what course (first, second, or third) you're editing first.

https://user-images.githubusercontent.com/20250366/111753866-5ae3b100-88bd-11eb-932e-f26646fc2c44.mp4

@amontes261

nishi7409 reviewed (changes requested):
Reviewing bc GitHub is barking at me about this.
https://github.com/nishi7409/HASSPathways/pull/33

amontes261 committed:
Finished Pathway Editing Feature

Changes:
	- Made it so icon is greyed-out when step 1, 2 or 3 is selected on the progress bar
	- Changed the normal non-selected icon state next to each course to a check instead of the original pencil
	- Made the selected-icon the pencil icon to signify editing

nishi7409 committed:
Merge branch 'master' into 2/Alex

nishi7409 commented:
- deletes pathway on same pathway added (through edit)
- progress bar is not stuck at the top of the page

nishi7409 reviewed (changes requested):
- deletes pathway on same pathway added (through edit)
- progress bar is not stuck at the top of the page

(github barked at me)

amontes261 committed:
Fixed Bugs with Pathway Editing Feature

- Resolved issue where trying to add existing pathway layout resulted in that pathway getting removed from the collection

- Resolved scrolling Issue with the "Editing Pathway" indicator

nishi7409 reviewed (approved):
finally, good code
